src/inmemory_cache.rs

```diff
@@ -6,19 +6,22 @@ use crate::{
     block::Block, db_state::SsTableId, filter::BloomFilter, flatbuffer_types::SsTableIndexOwned,
 };
 
+/// The cached block types.
 #[derive(Clone)]
-pub(crate) enum CachedBlock {
+pub enum CachedBlock {
     Block(Arc<Block>),
     Index(Arc<SsTableIndexOwned>),
     Filter(Arc<BloomFilter>),
 }
 
+/// The type of the in-memory cache.
 #[derive(Clone, Copy)]
```

Comment:
I think we should get rid of this since we're getting rid of `block_cache_options`.

Reply:
removed.